dashboard/app/controllers/application_controller.rb

```diff
@@ -26,6 +26,16 @@ def fix_crawlers_with_bad_accept_headers
     end
   end
 
+  if Rails.env.development?
+    # Enable or disable the rack mini-profiler if the 'pp' query string parameter is set.
+    # pp='disabled' will disable it; any other value will enable it.
+    before_filter :maybe_enable_profiler
+    def maybe_enable_profiler
+      pp = params['pp']
+      ENV['RACK_MINI_PROFILER'] = (pp == 'disabled') ? 'off' : 'on' if pp
```

Comment:
Nit: an `if` and ternary on the same line gets a little convoluted.  Cleaner to write as this?
```ruby
if pp
  ENV['RACK_MINI_PROFILER'] = (pp == 'disabled') ? 'off' : 'on'
end
```